dbc_extract3/dbc/data.py

```diff
@@ -643,6 +643,9 @@ def initialize_data_model(options):
         if options.build >= dbc.WowVersion(8, 2, 0, 30080):
             SpellName.link('azerite_essence', AzeriteEssencePower)
 
+        if options.build >= dbc.WowVersion(9, 0, 0, 0):
+            SpellName.link('targeting', SpellTargetRestrictions)
+
```

Comment:
If you go by the new data model stuff (see comment below where you add the field to spell_data_t record output), linking is completely unnecessary.